feeds/pypi/main.go

```diff
@@ -58,32 +77,47 @@ func fetchPackages() ([]*Package, error) {
 	return rssResponse.Packages, nil
 }
 
-// PubSubMessage is the payload of a Pub/Sub event.
-type PubSubMessage struct {
-	Data []byte `json:"data"`
-}
-
 // Poll receives a message from Cloud Pub/Sub. Ideally, this will be from a
 // Cloud Scheduler trigger every `delta`.
-func Poll(ctx context.Context, m PubSubMessage) error {
+func Poll(ctx context.Context, t *pubsub.Topic) error {
```

Comment:
This is the one thing I'm not sure about. I anticipated that this would work via a pubsub trigger that occurs from a cloud scheduler. Basically setting up the cloud scheduler [like this](https://cloud.google.com/scheduler/docs/tut-pub-sub) to run every 5 minutes, and then following [this tutorial](https://cloud.google.com/functions/docs/calling/pubsub) to trigger the cloud function. That's where the message signature came from, and the `main` was just for local testing.

I'm _far_ from a GCP expert so please correct me, but it looks like this adjusts things to be more of a long-lived process that pulls from a topic? Do we anticipate that we can use the changes you've made here to do the time-based execution I mentioned above?

Reply:
> I'm _far_ from a GCP expert so please correct me, but it looks like this adjusts things to be more of a long-lived process that pulls from a topic?

Ah, no. This topic is the thing we *write to* after processing the index.

We still need to hook up the thing that "triggers" this to run after some kind of cron.

Reply:
Fixed to keep the signature!